File: customer-manager-backend/src/main/java/com/wushu/customer/manager/model/DynamicTableRecord.java
package com.wushu.customer.manager.model;

import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;

import java.util.Date;
import java.util.Map;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class DynamicTableRecord {
    private Long id;
    private String tableKey;           // 表唯一标识
    private Map<String, Object> data;  // 动态字段数据，以JSON格式存储
    private Date createTime;           // 创建时间
    private Date updateTime;           // 更新时间
}

File: customer-manager-frontend/src/models/DynamicTableRecord.ts
// DynamicTableRecord.ts
export interface DynamicTableRecord {
  id?: number;
  tableKey: string;                    // 表唯一标识
  data: Record<string, any>;          // 动态字段数据，以JSON格式存储
  createTime?: string;                // 创建时间
  updateTime?: string;                // 更新时间
}
